client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -127,10 +127,7 @@ class EditorConfirmationSidebar extends Component {
 		const buttonLabel = this.getBusyButtonLabel( this.props.publishButtonStatus );
 
 		return (
-			<Button
-				disabled
-				className="editor-confirmation-sidebar__publishing-button is-busy is-primary"
-			>
+			<Button	disabled= { disabled } className="editor-confirmation-sidebar__publishing-button is-busy is-primary">			
```

Comment:
There's a whitespace between `disabled=` and `{ disabled }` which seem to cause trouble when publishing a post.

Reply:
Good spot, thanks!

Reply:
Ah, sorry.. by looking closer it's not the whitespace but that `disabled` is not defined at this place. Seems like this addition came in through 745d2e5

Reply:
It's probably also more reliable to use the `busy` and `primary` props for this button and not directly apply the classes to respect the `<Button />` component API.

Let's try this:

```jsx
<Button
  disabled
  primary
  busy
  className="editor-confirmation-sidebar__publishing-button"
>
  { buttonLabel }
</Button>

```

Reply:
Thanks for whipping that up!